src/components/v2/deployment-history/ea/EADeploymentHistory.component.tsx

```diff
@@ -298,18 +309,21 @@ function ExternalAppDeploymentHistory({ appId }: { appId: string }) {
         );
     }
 
-    function renderSelectedDeploymentDetailHeader(){
-        let deployment = deploymentHistoryArr[selectedDeploymentHistoryIndex];
-        return <div className="trigger-details ml-20 pb-20">
-            <div className="trigger-details__summary">
-                <div className="flex column left pt-10">
-                    <div className="cn-9 fs-14 fw-6">Deployed at</div>
-                    <div className="flex left">
-                        <time className="cn-7 fs-12">
-                            {moment(new Date(deployment.deployedAt.seconds * 1000), 'YYYY-MM-DDTHH:mm:ssZ').format(Moment12HourFormat)}
-                        </time>
-                        {
-                            deployment.dockerImages.map((dockerImage, index) => {
+    function renderSelectedDeploymentDetailHeader() {
+        const deployment = deploymentHistoryArr[selectedDeploymentHistoryIndex]
+
+        return (
+            <div className="trigger-details ml-20 mr-20 pb-20">
+                <div className="flex content-space trigger-details__summary">
```

Comment:
I assume adding class "flex content-space" is intentional

Reply:
Yes. It's to show the Deployment info & Deploy button in a row with some alignments/spacing.